Change Visual studio image link in README.md

Ashleyt28 opened the pull request:
The Visual Studio Logo is missing from README.md @Issue#3089. Since the source link of the image is rendering 404 as an error, I guess the page is not available anymore. 

I've changed the image source link with a new one.

![vs17](https://user-images.githubusercontent.com/23654823/36779889-e2edea9a-1c8a-11e8-97ef-fdc8aca7094d.png)

closes #3089
Best,
Ashley Thanacoody

Ashleyt28 committed:
Change Visual studio image link in README.md

The Visual Studio Logo is missing from README.md @Issue#3089. Since the source link of the image is rendering 404 as an error, I guess the page is not available anymore. 

I've changed the image source link with a new one.

sara-02 reviewed (commented):
@Ashleyt28 thanks.
Also, will you be adding changes to other translations README, or do you want to leave it as a part of https://github.com/Roshanjossey/first-contributions/issues/96.
This PR is good either way. I can merge it now, or wait for your updates on other READMEs.

Ashleyt28 committed:
Change Visual studio image link in README.ar.md

Ashleyt28 committed:
Change Visual studio image link in README.chs.md

Ashleyt28 committed:
Change Visual studio image link in README.cht.md

Ashleyt28 committed:
Update README.de.md

Ashleyt28 committed:
Change Visual studio image link in README.eg.md

Ashleyt28 committed:
Change Visual studio image link in README.es.md

Ashleyt28 committed:
Change Visual studio image link in README.fa.md

Ashleyt28 committed:
Change Visual studio image link in README.fa.md

Ashleyt28 committed:
Change Visual studio image link in README.fa.md #

Ashleyt28 committed:
Change Visual studio image link in README.fa.md

Ashleyt28 committed:
Change Visual studio image link in README.fr.md

Ashleyt28 committed:
Change Visual studio image link in README.gl.md

Ashleyt28 committed:
Change Visual studio image link in README.gr.md

Ashleyt28 committed:
Change Visual studio image link in README.hi.md

Ashleyt28 committed:
Change Visual studio image link in README.id.md

Ashleyt28 committed:
Change Visual studio image link in README.it.md

Ashleyt28 committed:
Change Visual studio image link in README.mm_unicode.md

Ashleyt28 committed:
Change Visual studio image link in README.nl.md

Ashleyt28 committed:
Change Visual studio image link in README.pl.md

Ashleyt28 committed:
Change Visual studio image link in README.pt_br.md

Ashleyt28 committed:
Change Visual studio image link in README.ro.md

Ashleyt28 committed:
Change Visual studio image link in README.ru.md

Ashleyt28 committed:
Change Visual studio image link in README.th.md

Ashleyt28 committed:
Change Visual studio image link in README.tr.md

Ashleyt28 committed:
Change Visual studio image link in README.ua.md

Ashleyt28 committed:
Change Visual studio image link in README.ur.md

Ashleyt28 committed:
Change Visual studio image link in README.vn.md

Ashleyt28 committed:
Change Visual studio image link in README.lt.md

Ashleyt28 committed:
Change Visual studio image link in README.ja.md

Ashleyt28 committed:
Change Visual studio image link in README.ko.md

Ashleyt28 commented:
@sara-02 , i changed the links for all translations, My bad I made a wrong commit for 7th and 9th commit, but I updated it with another commit.
Awaiting your feedback,
Cheers

rbstrachan commented:
@Ashleyt28 Thank you for picking this up. I have been rushed off my feet with uni work, I appreciate the help!

Ashleyt28 committed:
Add link in Useful-links-for-further-learning.md

Link is about pull request reviews

Ashleyt28 committed:
Update Useful-links-for-further-learning.md

Added a link concerning pull request reviews

Ashleyt28 committed:
Delete Useful-links-for-further-learning.md

Ashleyt28 commented:
@sara-02 , These are my last commit concerning the issue  @Issue #2690. I don't know if this okay for it to be in this pull request. 
Thanks for your understanding

Roshanjossey commented:
@Ashleyt28, changes for visual studio code looks great. I'm merging this pull request

Thank you so much for your contribution 😄
